packages/aws-ssm-secrets/action.yml

```diff
@@ -9,7 +9,7 @@ inputs:
     description: "The corresponding environment variable name to assign the secret to"
     required: true
 runs:
-  using: "node12"
+  using: "node16"
```

Comment:
this bit on the other 2 actions to match node16

Reply:
Updated @sbe-arg